students/chylak_malgorzata/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,20 @@
+#! python 3
+# deleting_unneeded_files walks through a folder tree and searches for
+# files larger than 100MB and prints their absolute path to the screen.
+
+import os
+
+
+def large_files(folder):
+    folder = os.path.abspath(folder)
+
+    for foldername, subfolders, filenames in os.walk(folder):
```

Comment:
According to PEP8 naming conventions variable name should looks like: lower_case_with_underscores